packages/SymbioticController/composer.json

```diff
@@ -4,7 +4,8 @@
         "php": "^7.1",
         "nette/di": "^2.4",
         "nette/application": "^2.4",
-        "odbav-to/presenter-route": "^0.1"
+        "odbav-to/presenter-route": "^0.1",
```

Comment:
What do you mean by **adding that class in here**?
It is not a single class anymore and since that we have made some improvements and new features.

To be honest it does not fit in here for me (the whole `PresenterRoute` package).

Imho you could do some basic and very simple implementation of route here and allow other packages to easily replace that implementation.

But in my eyes moving completely `odbav-to/presenter-route` here is not a good decision.